src/helix-ui/styles/elements/hx-drop-fence.less

```diff
@@ -0,0 +1,13 @@
+@import (reference) 'components/box';
+
+hx-drop-fence {
+  #Box.md();
+  border: 1px dashed @gray-600;
+  display: block;
+  font-size: 0.875rem;
+  text-align: center;
+
+  > *:first-child {
+    margin-top: 0.5rem;
```

Comment:
This doesn't work for text-node children `<hx-drop-fence>I'm text</hx-drop-fence>`